src/appkit.cs

```diff
@@ -2026,7 +2071,7 @@ interface NSButtonCell {
 	}
 	
 	[BaseType (typeof (NSControl))]
-	interface NSButton : NSAccessibilityButton {
+	interface NSButton : NSAccessibilityButton, NSUserInterfaceCompression {
```

Comment:
Looking at NSUserInterfaceCompression, it has abstract members.  Is that a breaking change to add to an existing class?